src/ty/mod.rs

```diff
@@ -271,9 +269,6 @@ pub enum TypeDef<T: Form = MetaForm> {
     /// A type representing a sequence of bits.
     #[codec(index = 7)]
     BitSequence(TypeDefBitSequence<T>),
-    /// A Range type.
-    #[codec(index = 8)]
-    Range(TypeDefRange<T>),
```

Comment:
Do you think we should make `TypeDef` be `#[non_exhaustive]`?

Reply:
I'm not sure because all `TypeDef` variants need to be handled usually, a wildcard might not make sense.